zhmcclient/_manager.py

```
@@ -672,6 +673,200 @@ def supports_properties(self):
+            Filter arguments that narrow the list of returned resources to
+            those that match the specified filter arguments. For details, see
+            :ref:`Filtering`.
+
+            `None` causes no filtering to happen, i.e. all resources are
+            returned.
+
+          additional_properties (list of string):
+            List of property names that are to be returned in addition to the
+            default properties.
+
+        Returns:
+
+          : A list of zhmcclient resource objects.
+
+        Raises:
+
+          :exc:`~zhmcclient.HTTPError`
+          :exc:`~zhmcclient.ParseError`
+          :exc:`~zhmcclient.AuthError`
+          :exc:`~zhmcclient.ConnectionError`
+        """
+        resource_obj_list = []
+        if self.auto_update_enabled() and not self.auto_update_needs_pull():
+            for resource_obj in self.list_resources_local():
+                if matches_filters(resource_obj, filter_args):
+                    resource_obj_list.append(resource_obj)
+        else:
+            resource_obj = self._try_optimized_lookup(filter_args)
+            if resource_obj:
+                resource_obj_list.append(resource_obj)
+                # It already has full properties
+            else:
+                query_parms, client_filters = divide_filter_args(
+                    self._query_props, filter_args)
+                if additional_properties:
+                    ap_parm = 'additional_properties={}'.format(
+                        ','.join(additional_properties))
+                    query_parms.append(ap_parm)
+                query_parms_str = make_query_str(query_parms)
+                uri = '{}{}'.format(list_uri, query_parms_str)
+
+                try:
+                    result = self.session.get(uri)
+                except HTTPError as exc:
+                    if exc.http_status == 404 and exc.reason == 1:
+                        # Unlike other list operations, "List Logical Partitions
```

404.1 is only expected for a specific case; maybe you want to check explicitly for that case, to avoid ignoring 404.1 on other operations?

---

Probably that would be good - checked via `isinstance(self, LparManager)`.

In addition, we need to re-raise the exception in all other cases, that was also missing. Fixed.